tests/15-rpl-classic/js/12-rpl-rank-error-mobility.dat

```diff
@@ -0,0 +1,7 @@
+0 0.0 0 0
+1 0.0 30 0
+2 0.0 60 0
+3 0.0 90 0
+3 180.0 30 20
+3 3600.0 30 20
+
```

Comment:
Is the blank line required?

Reply:
I have this as a habit to ensure that the last line has terminating symbols. The idea is that some systems/tools/etc. out there defines a line as chars ending with terminators, and may misbehave if not present. Can't say I have done extensive research on this, but quick googling seems to say it is reasonable - happy to be convinced otherwise.

Reply:
This particular file is read by Cooja which isn't really portable at all (Linux/OS X and maybe Windows/FreeBSD). Even if Cooja was portable, one would want a fast host to run simulations, and to run things at all the host still needs a few gigs of RAM and a JVM. I think this narrows down the host platforms to Linux-like/OS X/Windows.

I'm not sure what to search for based on your description, isn't EOF a terminator?

Without being sure what class of bugs you are trying to avoid with the empty line, my gut feeling for the general case is that pipelining basic unix tools (awk/sed/grep/perl) becomes harder with files that contain spurious blank lines since the blank lines need to be filtered out from the main loop to not end up with an empty variable for the line (`$_` in perl, I have no idea what it is called in other languages).

Reply:
Sorry @pjonsson, I see now that I have added an actual blank line (as you of course wrote in the first comment). So there is now two LFs at the end of the file. This was not my intention, I wanted to ensure the last line had an LF. I got confused because in my local editor, if the last line has a LF, a blank line is shown beneath it. When I looked at the file here in github.com I saw a blank line and thought everything was as I wanted, but did not realize github does not show a separate blank line when the last line has LF - it was actually a second LF there.

> I'm not sure what to search for based on your description, isn't EOF a terminator?

I don't believe it is no, leaning on SO: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
I have removed the stray line